File: 125-valid-palindrome/125-valid-palindrome.go
func isPalindrome(s string) bool {
	var p string
	for _, v := range s {
		if v >= 65 && v <= 90 {
			p += string(v + 32)
		} else if v >= 97 && v <= 122 || v >= 48 && v <= 57 {
			p += string(v)
		}
	}

	for i := 0; i < len(p); i++ {
		if p[i] != p[len(p)-i-1] {
			return false
		}
	}

	return true
}

File: 125-valid-palindrome/125-valid-palindrome.js
/**
 * @param {string} s
 * @return {boolean}
 */
var isPalindrome = function (s) {
  let r = '';
  for (let i = 0; i < s.length; i++) {
    const code = s[i].charCodeAt(0);
    if (code >= 65 && code <= 90) {
      r += s[i].toLowerCase();
    } else if (code >= 97 && code <= 122 || code >= 48 && code <= 57) {
      r += s[i];
    }
  }

  return r === r.split('').reverse().join('');
};